packages/core/helper-plugin/src/hooks/tests/useFocusWhenNavigate.test.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable check-file/filename-naming-convention */
+
+import { lightTheme, ThemeProvider, Main } from '@strapi/design-system';
+import { render, renderHook, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import { useFocusWhenNavigate } from '../useFocusWhenNavigate';
+
+const consoleWarnMock = jest.spyOn(console, 'warn').mockImplementation();
+
+const setup = () => {
+  return render(
+    <ThemeProvider theme={lightTheme}>
+      <IntlProvider locale="en" messages={{}} defaultLocale="en">
```

Comment:
I believe you can remove these, similar to the suggestion on:

https://github.com/strapi/strapi/pull/17992/files#r1321089293